### PR TITLE
Upgrade demux-postgres

### PR DESCRIPTION
**Pull request by flux627**
- Upgrade linting, demux, demux-postgres, and massive
- Set type on massiveInstance for MassiveEosActionReader
- Update typedoc for MassiveEosActionReader#matchActionType
- Make necessary changes to fix new linting rules

### CODE REVIEW

File: src/readers/mongo/MongoActionReader.ts [hunk 1]
```diff
@@ -96,10 +105,6 @@ export class MongoActionReader extends AbstractActionReader {
   }
 
   protected async setup(): Promise<void> {
-    if (this.initialized) {
```
Comment (jeffreyssmith2nd) [inline src/readers/mongo/MongoActionReader.ts:99]: Why this change?

Comment (flux627) [reply src/readers/mongo/MongoActionReader.ts:99]: Ah, forgot to mention that. We're already doing this in the parent method `initialize`, so there's no need to here.

Comment (jeffreyssmith2nd) [reply src/readers/mongo/MongoActionReader.ts:99]: 👍

**Review by jeffreyssmith2nd — APPROVED** [review]
